python-spec/src/somacore/data.py

```diff
@@ -66,6 +67,10 @@ def create(
             index columns (e.g., ``['cell_type', 'tissue_type']``).
             All named columns must exist in the schema, and at least one
             index column name is required.
+        :param shape: The maximum capacity (domain) of the dataframe, including room
+            for any intended future appends. This must be in the postive int64
+            range, or `None`.  If `None` then the maximum possible int64 will be
```

Comment:
In docstrings, always use double quotes around raw unformatted text (i.e., no directives like `:class:`), so <code>\`\`None\`\`</code>, <code>`\`SOMADataFrame\`\`</code>, etc. (here and elsewhere). (This applies no matter the overall docstring format, whether standard Sphinx, NumPy, or Google style.)